src/matrix_multiple_mp_compare/matrix-multiply.cpp

```diff
@@ -0,0 +1,393 @@
+/***************************************************************************
+ *
+ *  Copyright (C) 2016 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  Codeplay's ComputeCpp SDK
+ *
+ *  matrix-multiply.cpp
+ *
+ *  Description:
+ *    Example of matrix multiplication in SYCL.
+ *
+ **************************************************************************/
```

Comment:
I'd asked Rafal to make some changes to the initial copyright header for the demos too, as some things don't make sense in this new repository:

```suggestion
/***************************************************************************
 *
 *  Copyright (C) Codeplay Software Limited
 *  Licensed under the Apache License, Version 2.0 (the "License");
 *  you may not use this file except in compliance with the License.
 *  You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 *  Unless required by applicable law or agreed to in writing, software
 *  distributed under the License is distributed on an "AS IS" BASIS,
 *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *  See the License for the specific language governing permissions and
 *  limitations under the License.
 *
 *  matrix-multiply.cpp
 *
 *  Description:
 *    Example of matrix multiplication in SYCL.
 *
 **************************************************************************/
```

Basically removing some lines that aren't needed, and removing the year from the copyright.

Reply:
Copyright text  changes have been made to both new examples.